storm-core/src/clj/org/apache/storm/daemon/executor.clj

```diff
@@ -742,7 +742,7 @@
                                                                (.getSourceComponent tuple)
                                                                (.getSourceStreamId tuple)
                                                                delta)))))))
-        has-eventloggers? (has-eventloggers? storm-conf)
+        has-eventloggers? (clojurify-structure (StormCommon/hasEventLoggers storm-conf))
```

Comment:
Here too I don't think `clojurify-structure` is needed.  Were you seeing errors without it?